tests/actions.test.js

```diff
@@ -0,0 +1,22 @@
+import { incrementCount } from '../src/redux/actions';
+import configureStore from 'redux-mock-store';
+
+const mockStore = configureStore();
+const store = mockStore();
+
+describe('test incrementCount action', () => {
+    beforeEach(() => {
+        store.clearActions();
+    });
+    it('dispatches correct action and payload', () => {
+        const expectedActions = [
+            {
+                'type': 'INCREMENT_COUNT',
+                'payload': { updated: (new Date()).valueOf() }
```

Comment:
Very unlikely to get timing issues now that the two times are created one line apart, but you could always do something like this to make sure you're using the same value:

```js
const current_time = new Date()
const expectedActions = [{'payload': {updated: current_time.valueOf()}}]
store.dispatch(incrementCount({updated: current_time.valueOf()}))
```

Reply:
Updated as per your suggestion just in case